Missing return types in typescript declaration file

<!--
*****************************************************************
* WARNING:  If you ignore this template, so too will your issue
*****************************************************************

  Provide a general summary of the issue in the Title above 
-->
## Your Environment
* Plugin version: 6.0.7
* Platform: Android
* OS version: 10
* Device manufacturer / model: OnePlus 7T
* Cordova version (`cordova -v`): 9.0.0 (cordova-lib@9.0.1)
* Cordova platform version (`cordova platform ls`):

## Expected Behavior
<!--- Tell us what should happen -->

Running a build with the `noImplicitAny` ([`tsconfig.json`](https://www.typescriptlang.org/docs/handbook/compiler-options.html)) option set to `true` should not throw any errors.

## Actual Behavior
<!--- Tell us what happens instead -->

Running a build throws this error:
```
> ng run app:ionic-cordova-build --platform=android
chunk {cordova} cordova.js, cordova.js.map (cordova) 63.5 kB [entry] [rendered]
chunk {main} main.js, main.js.map (main) 626 bytes [initial] [rendered]
chunk {polyfills} polyfills.js, polyfills.js.map (polyfills) 681 bytes [initial] [rendered]
chunk {runtime} runtime.js, runtime.js.map (runtime) 6.15 kB [entry] [rendered]
chunk {styles} styles.js, styles.js.map (styles) 236 kB [initial] [rendered]
Date: 2020-05-27T10:47:34.752Z - Hash: 3a7986393a7c65fdb1cc - Time: 25758ms
ERROR in node_modules/cordova-plugin-background-fetch/src/typescript/index.d.ts:163:10 - error TS7010: 'scheduleTask', which lacks return-type annotation, implicitly has an 'any' return type.
163     static scheduleTask(config:TaskConfig, success?:() => void, failure?:(error:string) => void);
               ~~~~~~~~~~~~
node_modules/cordova-plugin-background-fetch/src/typescript/index.d.ts:165:10 - error TS7010: 'stopTask', which lacks return-type annotation, implicitly has an 'any' return type.
165     static stopTask(taskId:string, success?:() => void, failure?:(error:string) => void);
               ~~~~~~~~
[ERROR] An error occurred while running subprocess ng.
        
        ng run app:ionic-cordova-build --platform=android exited with exit code 1.
        
        Re-running this command with the --verbose flag may provide more information.
```

## Steps to Reproduce
<!--- reproduce this issue; include code to reproduce, if relevant -->
1. Set `noImplicitAny` in `tsconfig.json` to `true`
2. `ionic cordova plugin add cordova-plugin-background-fetch`
3. `npx ionic build`

## Context
<!--- What were you trying to do? -->

## Debug logs
<!-- include iOS / Android logs
- ios XCode logs, 
- use #getLog #emailLog methods (@see docs)
- Android: $ adb logcat
-->

--- a/src/typescript/index.d.ts
+++ b/src/typescript/index.d.ts
@@ -160,9 +160,9 @@ declare module "cordova-plugin-background-fetch" {
 		*/
 		static start(success?:() => void, failure?:(status:BackgroundFetchStatus) => void):void;
 
-		static scheduleTask(config:TaskConfig, success?:() => void, failure?:(error:string) => void);
+		static scheduleTask(config:TaskConfig, success?:() => void, failure?:(error:string) => void):void;
 
-		static stopTask(taskId:string, success?:() => void, failure?:(error:string) => void);
+		static stopTask(taskId:string, success?:() => void, failure?:(error:string) => void):void;
 
 		/**
 		* Stop subscribing to fetch events.

